### PR TITLE
`Hello World!` in java

### PR DESCRIPTION
**Pull request by tempewda**

### CODE REVIEW

File: hello.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import java.util.*;
+
+public class Hello
+{
+  public static int main(String[] args)
```
Comment (LordUbuntu) [inline hello.java:5]: Is `String[] args` necessary in this case? The program takes no arguments.

**Comment by LordUbuntu** [comment]
I greatly appreciated both your clarity and the use of best practices (seperate branch, descriptive commits) in your contribution to this project! Thank you for your great work, it's been merged!

**Comment by LordUbuntu** [comment]
I greatly appreciated both your clarity and the use of best practices (seperate branch, descriptive commits) in your contribution to this project! Thank you for your great work, it's been merged!

File: hello.java (hunk 1 above)
Comment (tempewda) [reply hello.java:5]: Yes, `String[] args` is necessary for the main  function, kinda part of the syntax as mentioned [here](https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.1.4:~:text=Finally%2C%20after%20completion,declarations%20is%20acceptable%3A)

Comment (LordUbuntu) [reply hello.java:5]: Good ol' Java

Comment (tempewda) [reply hello.java:5]: my bad though, `main` should have been `void`, instead of `int`.

Comment (LordUbuntu) [reply hello.java:5]: No need to worry about anything.

**Pull request merged**